packages/uniextract2.vm/uniextract2.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>uniextract2.vm</id>
-    <version>2.0.0.5</version>
+    <version>2.0.0.20220113</version>
```

Comment:
This version is tricky. From Chocolatey documentation:

> So if the package was ruby and the version was 2.0.0-p353, the package is 2.0.0.35300 (adding the two zeroes at the end)

But there is one case that Chocolatey documentation doesn't cover.  As I understand the documentation `Ruby 2.7.0-preview3` would be `2.7.0.300` and `Ruby 2.7.0-rc1` (released after `Ruby 2.7.0-preview3`) would be `Ruby 2.7.0.100`, which would conflict with `Ruby 2.7.0-preview1`. Or I am understanding something wrong?

I have use the date format while we think how to resolve this. The previous version was already not following Chocolatey documentation (as the fourth segment needed two extra 0s).